scripts/build_doxygen.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
```

Comment:
:no_entry_sign: I think you need `set -euo pipefail` if you want the job to fail when the doxygen build returns non-zero.

Reply:
Sorry, I misread. The code is correct as the command is the last one in the script so the return value will be passed on. As discussed offline, it would still be preferable to add the pipefail command and "de-conjunctise" the last command.